### PR TITLE
update to use latest swagger

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/search/azure-search-documents/azure/search/documents/_generated/_serialization.py [hunk 1]
```diff
@@ -1910,7 +1912,7 @@ def deserialize_date(attr):
         if re.search(r"[^\W\d_]", attr, re.I + re.U):  # type: ignore
             raise DeserializationError("Date must have only digits and -. Received: %s" % attr)
         # This must NOT use defaultmonth/defaultday. Using None ensure this raises an exception.
-        return isodate.parse_date(attr, defaultmonth=None, defaultday=None)
+        return isodate.parse_date(attr, defaultmonth=0, defaultday=0)
```
Comment (kashifkhan) [inline sdk/search/azure-search-documents/azure/search/documents/_generated/_serialization.py:1915]: I know this is generated code, but Im guessing this change will not impact the exception behavior from the comment?

File: sdk/search/azure-search-documents/azure/search/documents/indexes/models/_index.py [hunk 2]
```diff
@@ -681,6 +683,8 @@ def pack_search_field(search_field: SearchField) -> _SearchField:
         synonym_map_names = search_field.get("synonym_map_names")
         fields = search_field.get("fields")
         fields = [pack_search_field(x) for x in fields] if fields else None
+        vector_search_dimensions = search_field.get("vector_search_dimensions")
+        vector_search_profile = search_field.get("vector_search_profile")
```
Comment (kashifkhan) [inline sdk/search/azure-search-documents/azure/search/documents/indexes/models/_index.py:687]: will this change end up being a breaking change or just internal rename ?

Comment (xiangyan99) [reply sdk/search/azure-search-documents/azure/search/documents/indexes/models/_index.py:687]: Yes. You find it. :)

This is a breaking change service introduced. The good news is that this is a  beta feature hence it is not breaking against last GA.

I would like to break it soon than later.

Comment (kashifkhan) [reply sdk/search/azure-search-documents/azure/search/documents/indexes/models/_index.py:687]: lol :) awesome, then other than a nit mention in the beta changelog, it looks fine

**Review by kashifkhan — APPROVED** [review]

**Pull request merged**